tests/lua/global_functions_test.lua

```diff
@@ -1,10 +1,28 @@
+local NIL_STRING = tostring(nil)
+
 -- Get global avatar instance
-local av = gapi.get_avatar()
+local avatar = gapi.get_avatar()
+
+-- Get avatar name
+local avatar_name = avatar.name
+local avatar_pos = avatar:get_pos_ms()
+
+-- Run some global functions
+local creature_avatar = gapi.get_creature_at(avatar_pos)
+local creature_avatar_name = if creature_avatar then creature_avatar:get_name() else NIL_STRING
```

Comment:
That's not valid Lua syntax. There's also no ternary operator in Lua, but it's functionality can be imitated like so:
```suggestion
local creature_avatar_name = creature_avatar and creature_avatar:get_name() or NIL_STRING
```

Reply:
Whoops, got too accustomed to Luau; fixed.